src/users/api/FullUserDto.ts

```diff
@@ -0,0 +1,50 @@
+import { ApiHideProperty, ApiProperty } from '@nestjs/swagger';
+import { Exclude } from 'class-transformer';
+
+export class FullUserDto {
```

Comment:
`Full UserDto` is a completed duplicate of `UserDto`, there is no difference between them except for some decorators. You have to use `groups` instead